orchagent/fdborch.cpp

```diff
@@ -1085,15 +1258,21 @@ bool FdbOrch::addFdbEntry(const FdbEntry& entry, const string& port_name,
     attr.id = SAI_FDB_ENTRY_ATTR_TYPE;
     if (fdbData.origin == FDB_ORIGIN_VXLAN_ADVERTIZED)
     {
-        attr.value.s32 = SAI_FDB_ENTRY_TYPE_STATIC;
+        attr.value.s32 =  SAI_FDB_ENTRY_TYPE_STATIC;
+    }
+    else if (fdbData.origin == FDB_ORIGIN_MCLAG_ADVERTIZED)
+    {
+        attr.value.s32 = (fdbData.type == "dynamic_local") ? SAI_FDB_ENTRY_TYPE_DYNAMIC : SAI_FDB_ENTRY_TYPE_STATIC;
     }
     else
     {
         attr.value.s32 = (fdbData.type == "dynamic") ? SAI_FDB_ENTRY_TYPE_DYNAMIC : SAI_FDB_ENTRY_TYPE_STATIC;
     }
+
     attrs.push_back(attr);
 
-    if ((fdbData.origin == FDB_ORIGIN_VXLAN_ADVERTIZED) && (fdbData.type == "dynamic"))
+    if ((fdbData.origin == FDB_ORIGIN_VXLAN_ADVERTIZED) || (fdbData.origin == FDB_ORIGIN_MCLAG_ADVERTIZED)
+            || (fdbData.type == "dynamic"))
```

Comment:
THIS modification does not look RIGHT.

Reply:
@lguohan : Thanks for pointing out. Thats right as part of WB, when the MAC addresses are added via the bake function, the attribute SAI_FDB_ENTRY_ATTR_ALLOW_MAC_MOVE is set wrongly. 
Will raise a PR and fix this today. Thanks.